libsolidity/codegen/MultiUseYulFunctionCollector.h

```diff
@@ -29,6 +29,16 @@
 namespace solidity::frontend
 {
 
+struct YulUtilFunction
+{
+	std::string name;
+	std::vector<std::string> parameters;
+	std::vector<std::string> returnValues;
```

Comment:
renamed `parameters` to `arguments` as well

Reply:
Well, I suggested `arguments` only in case you wanted to actually keep `returnValues`. I prefer `parameters` + `returnParameters` for symmetry. Though this is not really all that important. Sorry for nitpicking this :)